CalPatRec/fcl/prolog.fcl

```diff
@@ -343,8 +343,9 @@ CalPatRec : { @table::CalPatRec
             goodHitMask                 : [ "TimeDivision" ]    ## is there a point in using combohits made
                                                                 ## out of straw hits with an apriory wrong wdist ?
             bkgHitMask                  : [  ]                  ##
-            writeFilteredComboHits      : 0                     ##
-            writeStrawHitFlags          : 1                     ## used by the track fit
+            writeFilteredComboHits      : 0                     ## by default, clone all hits with updated flags
```

Comment:
Are these parameters boolean?  If so, why not initialize with 'true' or 'false'?

Reply:
see below (:smile:)